DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/add-oneapi/README.md

```diff
@@ -0,0 +1,126 @@
+# `Platform Designer` Sample - oneAPI code
```

Comment:
I suggest that we remove this README (therefore I'm not reviewing it now)

Reply:
I prefer to keep for completeness